scripts/docker/docker-compose.yml

```diff
@@ -0,0 +1,103 @@
+version: "3.7"
+
+networks:
+  dcf-network:
+    driver: bridge
+
+services:
+  mongodb:
+    container_name: dcf-mongodb
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: dbAdmin
+      MONGO_INITDB_ROOT_PASSWORD: password
+    hostname: dcf-mongo-db
+    image: mongo:4.4-focal
+    networks:
+      dcf-network: { }
+    ports:
+      - "27017:27017/tcp"
+    restart: always
+    volumes:
+      - /var/lib/mongo
+
+  arangodb:
+    container_name: dcf-arango
+    environment:
+      ARANGO_NO_AUTH: 1
+    hostname: dcf-arango-db
+    image: arangodb:3.8
+    networks:
+      dcf-network: { }
+    ports:
+      - "8529:8529/tcp"
+    restart: always
+    volumes:
+      - /var/lib/arango
+
+  mqtt-broker:
+    command: "/usr/sbin/mosquitto -c /mosquitto-no-auth.conf"
+    container_name: dcf-mqtt-broker
+    hostname: dcf-mqtt-broker
+    image: eclipse-mosquitto:2.0
+    networks:
+      dcf-network: { }
+    ports:
+      - "0.0.0.0:1883:1883/tcp"
+    read_only: true
+    restart: always
+
+  policy-agent:
+    command: "run --server /policies/code.rego /policies/data.json"
+    container_name: dcf-policy-agent
+    hostname: dcf-policy-agent
+    image: openpolicyagent/opa:0.36.0
+    networks:
+      dcf-network: { }
+    ports:
+      - "8181:8181/tcp"
+    restart: always
+    volumes:
+      - ../policies:/policies
+
+  dcf-subscriber:
+    container_name: dcf-subscriber
+    depends_on:
+      - arangodb
+      - mqtt-broker
+    image: octo-dcf/scoring-apps-go/docker-subscriber-go:0.0.0-dev
```

Comment:
Trying to pull this image gives the following error:
 `ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.` After choosing to continue, I get this error:
 `ERROR: pull access denied for octo-dcf/scoring-apps-go/docker-subscriber-go, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`
There might be different reasons for this. I am logged in with my personal account, so is this on a private registry? Please tell me if I am missing out on something.

Reply:
Did you execute the `make docker` step first? That should build the images locally and make them available. The subscriber image above should be resident on the local machine (assuming it's been built) and no need to pull from remote.

Reply:
That resolved the error, thank you! Do you think it would be useful if we include that step in the README?